cli/src/main/java/org/jboss/as/cli/logger/CliLogger.java

```diff
@@ -0,0 +1,235 @@
+    @Message(id = BASE + 40, value = "Exception while retrieving ls description properties", format = Message.Format.MESSAGE_FORMAT)
+    void exceptionRetrievingDescriptionProperties(@Cause Exception cause);
+
+    @LogMessage(level = Logger.Level.TRACE)
+    @Message(id = BASE + 41, value = "Access-control is {0}", format = Message.Format.MESSAGE_FORMAT)
+    void accessControlStatus(boolean status);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = BASE + 42, value = "Unable to obtain default configuration", format = Message.Format.MESSAGE_FORMAT)
+    void cannotObtainDefaultConfiguration(@Cause Exception cause);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = BASE + 43, value = "Failed to handle '{0}'", format = Message.Format.MESSAGE_FORMAT)
+    void failedToHandleLine(String line, @Cause Throwable cause);
+
+    @LogMessage(level = Logger.Level.ERROR)
+    @Message(id = BASE + 44, value = "Print message error: {0}", format = Message.Format.MESSAGE_FORMAT)
+    void printMessageError(String message);
+
+    @LogMessage(level = Logger.Level.INFO)
+    @Message(id = BASE + 45, value = "Print message: {0}", format = Message.Format.MESSAGE_FORMAT)
+    void printMessageInfo(String message);
+
+    @LogMessage(level = Logger.Level.INFO)
+    @Message(id = BASE + 46, value = "Print columns: {0}", format = Message.Format.MESSAGE_FORMAT)
+    void printColumns(Collection<String> columns);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = BASE + 47, value = "Connecting to {0} as {1}", format = Message.Format.MESSAGE_FORMAT)
+    void connectingToHostAs(String address, String userName);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = BASE + 48, value = "Invalid response getting candidates for the deployment", format = Message.Format.MESSAGE_FORMAT)
+    void invalidResponseForDeploymentCandidates();
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = BASE + 49, value = "Exception getting candidates for the deployment", format = Message.Format.MESSAGE_FORMAT)
+    void exceptionGettingCandidates(@Cause Exception cause);
+
+    @LogMessage(level = Logger.Level.WARN)
+    @Message(id = BASE + 50, value = "Error parsing line: {0}", format = Message.Format.MESSAGE_FORMAT)
+    void errorParsingLine(String message, @Cause Exception cause);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = BASE + 51, value = "Error when parsing last chunk of operation request", format = Message.Format.MESSAGE_FORMAT)
+    void errorParsingLastChunk(@Cause Exception cause);
+
+    @LogMessage(level = Logger.Level.TRACE)
+    @Message(id = BASE + 52, value = "Error getting capability names", format = Message.Format.MESSAGE_FORMAT)
+    void errorGettingCapabilityNames(@Cause Exception cause);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = BASE + 53, value = "Executing operation request:\n{0}", format = Message.Format.MESSAGE_FORMAT)
```

Comment:
This should be changed to:
```java
@Message(id = 153, value = "Executing operation request:%n%s")
```

Reply:
To clarify what I mean here is we shouldn't be using `\n` as on Windows this won't print as a new line. Using the `MessageFormat.PRINTF` and `%n` will make it platform independent.

Reply:
@jamezp Thanks for the comments, I will update the PR